build/scroller.d.ts

```diff
@@ -0,0 +1,139 @@
+type Align = "center";
+
+type Anchors = "hidden" | "visible";
+
+type Scrollbar = "hidden" | "visible";
+
+interface ScrollerConfig<Element extends HTMLElement> {
+  align?: Align;
```

Comment:
I found only `center` and undefined in the codebase

Do you need other values?

Reply:
Update: also added `left` and `right`

But from code perspective it doesn't do anything with it and equals to `undefined`

Reply:
I will add the JSDOC about default value

Reply:
Added ✅